docs/src/main/sphinx/release/release-443.md

```diff
@@ -0,0 +1,42 @@
+# Release 443 (21 Mar 2024)
+
+## General
+
+* Ensure consistent formatting when casting from double or real to varchar. ({issue}`21136`)
+* Prevent query failure when partial aggregation over long decimals is pushed
+  below joins. ({issue}`21099`)
+
+## Delta Lake connector
+
+* Support automatic type coercion for timestamp type columns during in table
+  creation. ({issue}`19336`)
+* Ensure deletion of blobs ending in slash when deleting directory contents and
```

Comment:
@electrum and @martint .. do we need to consider have a separate section for all native file systems .. or one for each file system ..  we could make all the release notes headers to be links to the relevant docs to help with that